documentation/Installation.md

```diff
@@ -1437,6 +1438,63 @@ services:
     cgroupDriver: systemd
 ```
 
+#### kubeadm_patches
+
+*Installation task*: `deploy.kubernetes`
+
+*Can cause reboot*: no
+
+*Can restart service*: always yes, `kubelet`
+
+*OS specific*: No
+
+In `services.kubeadm_patches` section you can override control-plane pod settings as well as kubelet settings on per node basis.
+This feature is implemented by using of [kubeadm patches](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/#patches).
+
+**Note**: `patches` feature is available for control-plane pods (etcd, kube-apiserver, kube-controller-manager, kube-scheduler) since Kubernetes **v1.22.0** and for kubelet since Kubernetes **v1.25.0**.
+
+The following is an example of control-plane settings override:
+```
+services:
+  kubeadm_patches:
+    apiServer:
+      - groups: [control-plane]
+        patch:
+          max-requests-inflight: 500
+      - nodes: [master-3]
+        patch:
+          max-requests-inflight: 600
+    etcd:
+      - nodes: [master-1]
+        patch:
+          snapshot-count: 110001
+      - nodes: [master-2]
+        patch:
+          snapshot-count: 120001
+      - nodes: [master-3]
+        patch:
+          snapshot-count: 130001
+    controllerManager:
+      - groups: [control-plane]
+        patch:
+          authorization-webhook-cache-authorized-ttl: 30
```

Comment:
The pod "kube-controller-manager" will not start with an error
`Error: invalid argument "30" for "--authorization-webhook-cache-authorized-ttl" flag: time: missing unit in duration "30"`

Should be changed to "30s"
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/

Reply:
IN updated.